inc/Engine/Optimization/RUCSS/Strategy/Strategies/DefaultProcess.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace WP_Rocket\Engine\Optimization\RUCSS\Strategy\Strategies;
+
+use WP_Rocket\Engine\Common\Clock\WPRClock;
+use WP_Rocket\Engine\Optimization\RUCSS\Database\Queries\UsedCSS as UsedCSS_Query;
+
+/**
+ * Class managing the default error for retry process of RUCSS
+ */
+class DefaultProcess implements StrategyInterface {
+	/**
+	 * UsedCss Query instance.
+	 *
+	 * @var UsedCSS_Query
+	 */
+	protected $used_css_query;
+
+	/**
+	 * Clock Object.
+	 *
+	 * @var WPRClock
+	 */
+	protected $clock;
+
+	/**
+	 * Represents a timetable which shows how long to wait after for a new retry depending on how many retries have been made already.
+	 *
+	 * @var int[]
+	 */
+	private $time_table_retry = [
+		1 => 180,  // 3 minutes
```

Comment:
I think we should keep a behavior close to the current one at the beginning (so 1 check every minute). So maybe:
1 => 60,
2 => 120,
3 => 300,
4 => 600,
5 => 1200
6 => 1800
?
(cc @piotrbak )

Reply:
I think too. I've based my initial numbers with the brief of the issue. I've adapted it back to the numbers you just give.